.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
+
+on:
+  pull_request:
+
+jobs:
+  build-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build
+        uses: diplodoc-platform/docs-build-action@v1
+        with:
+          revision: "pr-${{ github.event.pull_request.number }}"
+          project-name: ${{ vars.DIPLODOC_PROJECT_NAME }}
+          src-root: "./"
```

Comment:
documentation located under the `docs` scope

https://github.com/diplodoc-platform/documentation-template/tree/main/docs